src/properties/property-effects.html

```diff
@@ -1523,11 +1508,11 @@
        */
       _invalidateProperties() {
         if (this.__dataInitialized) {
-          if (this._asyncEffects) {
-            super._invalidateProperties();
-          } else {
+          // if (this._asyncEffects) {
```

Comment:
Remove commented code. Add note: propertyAccessors takes changes async and propertyEffects does not.